doc/classes/LinkButton.xml

```diff
@@ -51,6 +51,9 @@
 		<member name="underline" type="int" setter="set_underline_mode" getter="get_underline_mode" enum="LinkButton.UnderlineMode" default="0">
 			Determines when to show the underline. See [enum UnderlineMode] for options.
 		</member>
+		<member name="url" type="String" setter="set_url" getter="get_url" default="&quot;&quot;">
+			If an URL is supplied, it will be opened on click.
+		</member>
```

Comment:
This barely has any documentation compared to https://github.com/godotengine/godot/pull/30675. Either the documentation should be copied or the other pr should be merged first.